src/datepicker/styled-components.js

```diff
@@ -36,7 +37,15 @@ export const StyledRoot = styled<SharedStylePropsT>('div', props => {
   };
 });
 
-export const StyledMonthContainer = styled('div', {display: 'flex'});
+export const StyledMonthContainer = styled<{
+  $orientation: $PropertyType<CalendarPropsT<Date>, 'orientation'>,
```

Comment:
is there a better way to type `$orientation`?

Reply:
This is fine with me. Another option may be to pull orientation type definition out to its own type and import that to this file
https://github.com/uber/baseweb/blob/master/src/datepicker/types.js#L177